.devcontainer/devcontainer.json

```diff
@@ -34,6 +34,11 @@
             "source": "overlay-${devcontainerId}",
             "target": "/opt/overlay_ws",
             "type": "volume"
+        },
+        {
+            "source": "nav2-bashhistory",
+            "target": "/commandhistory",
```

Comment:
When I overhaul the user account setup for the devcontainer, we can track such dot files easier than using root level paths.